src/jmh/java/hu/akarnokd/comparison/scrabble/ShakespearePlaysScrabbleWithCyclopsReactOpt.java

```diff
@@ -49,145 +66,99 @@
     )
     @Fork(1)
     public List<Entry<Integer, List<String>>> measureThroughput() throws InterruptedException {
-
-        //  to compute the score of a given word
-        Function<Integer, Integer> scoreOfALetter = letter -> letterScores[letter - 'a'];
-
-        // score of the same letters in a word
-        Function<Entry<Integer, MutableLong>, Integer> letterScore =
-                entry ->
-                        letterScores[entry.getKey() - 'a'] *
-                        Integer.min(
-                                (int)entry.getValue().get(),
-                                scrabbleAvailableLetters[entry.getKey() - 'a']
-                            )
-                    ;
-
-
-        Function<String, ReactiveSeq<Integer>> toIntegerIx =
-                string -> chars(string);
-
-        // Histogram of the letters in a given word
-        Function<String, ReactiveSeq<HashMap<Integer, MutableLong>>> histoOfLetters =
-                word ->  toIntegerIx.apply(word)
-                            .scanLeft(new HashMap<Integer, MutableLong>(), (map, value) -> {
-                                MutableLong newValue = map.get(value) ;
-                                if (newValue == null) {
-                                    newValue = new MutableLong();
-                                    map.put(value, newValue);
-                                }
-                                newValue.incAndSet();
-                                return map;
-                            })
-                        .takeRight(1);
-                 ;
-
-        // number of blanks for a given letter
-        Function<Entry<Integer, MutableLong>, Long> blank =
-                entry ->
-                        Long.max(
-                            0L,
-                            entry.getValue().get() -
-                            scrabbleAvailableLetters[entry.getKey() - 'a']
-                        )
-                    ;
-
+                                               
+         // Histogram of the letters in a given word
+        Function<String, ReactiveSeq<Map.Entry<Integer, Long>>> histoOfLetters =
```

Comment:
Please don't inline the functions but keep them as is because they form the basis of comparing the same steps with the original.